pkg/agent/bakerapi_test.go

```diff
@@ -184,11 +184,19 @@ var _ = Describe("AgentBaker API implementation tests", func() {
 
 		It("should return the correct bootstrapping data when linux node image version override is present", func() {
 			toggles.Maps = map[string]agenttoggles.MapToggle{
-				"linux-node-image-version": func(entity *agenttoggles.Entity) map[string]string {
+				"vhd-types": func(entity *agenttoggles.Entity) map[string]string {
 					return map[string]string{
-						string(datamodel.AKSUbuntu1604): "202402.27.0",
+						"vhd-type": "override",
```

Comment:
this would be `1P` in real cases right?

Reply:
It would have been, but I reverted this change. Didn't want to leak 1p vs 3p terms into this repo.